optimize/self-managed/optimize-deployment/configuration/security-instructions.md

```diff
@@ -59,11 +59,11 @@ Authorization controls what data a user can access and change in Optimize once a
 
 </TabItem>
 
-<TabItem value='elasticsearch'>
+<TabItem value='database'>
 
-Optimize stores its data in Elasticsearch, which is a search engine that acts as a document-based datastore. To protect access to this data, Elasticsearch should be configured carefully as well. Refer to the official [Secure the Elastic Stack](https://www.elastic.co/guide/en/elasticsearch/reference/master/secure-cluster.html#secure-cluster) documentation of Elasticsearch.
+Optimize stores its data in Elasticsearch or OpenSearch, which is a search engine that acts as a document-based datastore. To protect access to this data, the database should be configured carefully as well. Refer to the official security guidelines for [ElasticSearch](https://www.elastic.co/guide/en/elasticsearch/reference/master/secure-cluster.html#secure-cluster) or [OpenSearch](https://opensearch.org/docs/latest/getting-started/security).
```

Comment:
`Optimize stores its data in Elasticsearch or OpenSearch, which is a search engine that acts as a document-based datastore`

The grammar on the latter half of the sentence needs changed

`Optimize stores its data in Elasticsearch or OpenSearch, which are search engines that act as a document-based datastore`